webservices/tasks/__init__.py

```diff
@@ -29,8 +29,12 @@ def redis_url():
     CELERYBEAT_SCHEDULE={
         'refresh': {
             'task': 'webservices.tasks.refresh.refresh',
-            'schedule': crontab(minute=0, hour=9),
+            'schedule': crontab(minute=0, hour=9, day_of_week='sun-fri'),
         },
+        'refresh_all': {
```

Comment:
Whoops, should probably change this to `refresh_and_rebuild` too - when I saw it come up in the logs I was like hey wait a minute...!  Oh. :-)